chain/cmd/bandsv/main.go

```diff
@@ -263,28 +264,62 @@ func handleExecute(c *gin.Context) {
 		return
 	}
 
-	var answer []string
-	for _, command := range commands {
-		if !allowedCommands[command.Cmd] {
-			c.JSON(http.StatusBadRequest, gin.H{"error": fmt.Errorf("handleRequest unknown command %s", command.Cmd)})
-			return
-		}
-		dockerCommand := Command{
-			Cmd: "docker",
-			Arguments: append([]string{
-				"run", "--rm", "band-provider",
-				command.Cmd,
-			}, command.Arguments...),
-		}
-		query, err := execWithTimeout(dockerCommand, 10*time.Second)
-		if err != nil {
-			c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+	type queryParallelInfo struct {
+		index      int
+		answer     string
+		httpStatus int
+		err        interface{}
+	}
+	chanQueryParallelInfo := make(chan queryParallelInfo)
+	for i, command := range commands {
+		go func(index int, command Command) {
```

Comment:
This can leak: it is possible that some goroutines will stuck and waste resource forever